include/deal.II/base/tensor.h

```diff
@@ -1356,24 +1337,9 @@ operator- (const Tensor<0,dim,Number> &p, const Tensor<0,dim,OtherNumber> &q)
  * Multiplication of a tensor of general rank with a scalar number from the
  * right.
  *
- * The purpose of this operator is to enable only multiplication of a tensor
- * by a scalar number (i.e., a floating point number, a complex floating point
- * number, etc.). The function is written in a way that only allows the
```

Comment:
I have removed the discussion about the return type because we have it everywhere in this header file - instead I put a lot of ```@relates``` in place. If one is interested it is possible to simply click on the the return type to get a description.

Further, the discussion about ```EnableIfScalar``` is much more informative on the class itself - this is the reason why I shortened it. I have put the same expression in place for the other multiplication.